src/crimson/os/seastore/segment_manager_group.cc

```diff
@@ -27,6 +27,7 @@ SegmentManagerGroup::read_segment_tail(segment_id_t segment)
   ).safe_then([=, &segment_manager](bufferptr bptr) -> read_segment_tail_ret {
     LOG_PREFIX(SegmentManagerGroup::read_segment_tail);
     DEBUG("segment {} bptr size {}", segment, bptr.length());
+    INFO("segment size = {}", segment_manager.get_segment_size());
```

Comment:
Seems to me it is an unrelated log about `read_segment_tail()`

Reply:
Yes, removed it.